src/highdicom/sc/sop.py

```diff
@@ -125,43 +126,43 @@ def __init__(
             Name of the manufacturer of the device that creates the SC image
             instance (in a research setting this is typically the same
             as `institution_name`)
-        patient_id: str, optional
+        patient_id: Union[str, None], optional
            ID of the patient (medical record number)
-        patient_name: Optional[Union[str, PersonName]], optional
+        patient_name: Union[str, PersonName, None], optional
            Name of the patient
-        patient_birth_date: str, optional
+        patient_birth_date: Union[str, None], optional
            Patient's birth date
-        patient_sex: str, optional
+        patient_sex: Union[str, highdicom.enum.PatientSexValues, None], optional
```

Comment:
We may not want to include the "private" `enum` namespace in the docstring.

```suggestion
        patient_sex: Union[str, highdicom.PatientSexValues, None], optional
```

Reply:
As noted below, this is because the enum sub-package is not exposed at the root of the package

Reply:
Resolved in 9a6f15f